services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -624,6 +631,46 @@ private void sendAdminMessageAndWaitForConsumed(String clusterName, String store
     waitingMessageToBeConsumed(clusterName, storeName, message.executionId);
   }
 
+  @Override
+  public boolean deleteValueSchemas(String clusterName, String storeName, Set<Integer> inuseValueSchemaIds) {
```

Comment:
Shall we enforce the unused schema id check here? Allowing random value schema deletion is very risky.